subxt/Cargo.toml

```diff
@@ -112,6 +112,10 @@ getrandom = { workspace = true, optional = true }
 # Included if "native" feature is enabled
 tokio-util = { workspace = true, features = ["compat"], optional = true }
 
+# Included if the reconnecting rpc client feature is enabled
+tokio = { workspace = true, optional = true }
```

Comment:
Clean up tokio, perhaps we can replace it with futures channels and stuff

Reply:
We'll prob want a tokio feature flag or similar anyway so that we can make it easy to start up the `UnstableBackend` etc!

I see `web` enabled `tokio?/sync; would `web` and `tokio/sync` work together though?

Reply:
yes, tokio/sync works for wasm/web but that could easily be replaced by futures channel

Reply:
I forgot that I used `tokio::sync::Notify` so I prefer to keep tokio but not runtime agnostic but it's doesn't matter anyway because jsonrpsee is already using tokio channels.

We could change that though if becomes an issue.

Reply:
Yup I don't mind that it uses tokio since jsonrpsee does anyway, so we don't really gain anything by trying to remove it anyways from this I guess!